fr.free.Qccrypt.appdata.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2023 Philippe Beaureilles -->
+<component type="desktop-application">
+  <id>fr.free.Qccrypt</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0+</project_license>
```

Comment:
This is not a valid spdx license ID.

https://spdx.dev/licenses/

Reply:
Ok, I will change it.